autoload/fzf/vim.vim

```diff
@@ -1320,6 +1322,7 @@ function! s:given_range(line1, line2)
   return []
 endfunction
 
+" [[args], fullscreen-boolean, [git-log-args]]
```

Comment:
fullscreen boolean should be the last argument by convention.

Reply:
Oh good to know.

Then I would move `<q-args>` to the front so that it can be easily separated before the rest is passed to `s:fzf`. What do you think?